validation/jinja2/validation/advanced_search.html

```diff
@@ -0,0 +1,56 @@
+{#
+    # Displays search form to perform an advanced search
+#}
+   
+{% extends 'validation/_base.html' %}
+{% import 'validation/_macros.html' as macros %}
+
+{% block content %}
+<script>
+    function getSelectedSpeakers() {
+        var items = document.getElementsByName('speaker-options')
+        var selected_speakers = "";
+        for (var i=0; i<items.length; i++){
+            if(items[i].type=='checkbox' && items[i].checked==true)
+                selected_speakers += items[i].value+",";
+        }
+
+        if (selected_speakers.length == 0) {
+            document.getElementById('speaker').value = 'None';
+        } else {
+            document.getElementById('speaker').value = selected_speakers;
+        }
+    }
+</script>
+<form method="GET" action="{{
+    url('validation:advanced_search_results')
+  }}">
+    Nehiyaw phrase:
+    <input class="form-control mr-sm-2" name="transcription" type="search" placeholder="Search" aria-label="Search"><br>
+    English phrase:
+    <input class="form-control mr-sm-2" name="translation" type="search" placeholder="Search" aria-label="Search"><br>
+    Analysis:
+    <input class="form-control mr-sm-2" name="analysis" type="search" placeholder="Search" aria-label="Search"><br>
+
+    Status:
+    <input type="radio" id="all" name="status" value="all" checked>
+    <label for="all">All</label>  
+    <input type="radio" id="validated" name="status" value="validated">
+    <label for="validated">Validated</label>  
+    <input type="radio" id="unvalidated" name="status" value="unvalidated">
+    <label for="unvalidated">Unvalidated</label> <br>
+
+    Speaker:
+    <input type="checkbox" id="all_speakers" name="speaker-options" value="all">
+    <label for="all_spakers">All</label><br>
+        {% for speaker in speakers %}
+        <input type="checkbox" id="{{ speaker }}" name="speaker-options" value="{{ speaker }}">
+        <label for="{{ speaker }}">{{ speaker }}</label><br>
+        {% endfor %}
+
+    <input type="hidden" name="speaker" id="speaker">
```

Comment:
There's probably some fancy Boostrap way of getting the alignment of all of the elements just right, but for this PR, this will do 😅